test/core.spec.js

```diff
@@ -232,4 +232,18 @@
     });
   });
 
+  describe('vulnerabilities', function() {
+    it('has no prototype pollution vulnerability', function() {
+      const malicious = `this.constructor.constructor('return process')().mainModule.require('child_process').execSync('curl 127.0.0.1')`;
+      Object.prototype.MAL_CODE= [ malicious ];
+      const compiled = dust.compile('{username} is an important person.{~n}');
+      const tmpl = dust.loadSource(compiled);
+      dust.render(tmpl, { username: 'Jane Doe' }, (err, output) => {
```

Comment:
how did we test this? 
what should be the result of this test without the above fix

Reply:
It will execute `curl 127.0.0.1` and the test fails